diagnostic.md

```diff
@@ -14,7 +17,8 @@ list items (`li`) using jQuery? This is intentionally vague. (Assume starting
 at the `OL` DOM node).
 
 <!-- your answer starts here -->
-
+$('li').each('body');
+$('li').on('body')
```

Comment:
From `<ol>`, two possible ways to select `<li>` elements are `$('ol > li');` and `$('ol').children();`.  Some other possible ways are `$('li');`, `$('body').find('li');`, and `$('body').children('ol').children('li');`.